### PR TITLE
[Release] Update index.json for extension [ aks-preview ] 

### PR DESCRIPTION
**Pull request by azclibot**
[Release] Update index.json for extension [ aks-preview ] 

 Triggered by Azure CLI Extensions Release Pipeline - ADO_BUILD_ID=327434

Last commit against master: https://github.com/Azure/azure-cli-extensions/commit/d5baa785ce337b3334fa0c070c509dbe2e532120

### CODE REVIEW

**Comment by karataliu** [comment]
cc @yungezz @arrownj

**Comment by karataliu** [comment]
can someone trigger rerun and merge this? thank you

**Comment by yonzhan** [comment]
add to S167

File: src/index.json [hunk 1]
```diff
@@ -275,6 +275,51 @@
...
+                "filename": "aks_preview-0.4.38-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.minCliCoreVersion": "2.0.49",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 2",
```
Comment (yungezz) [inline src/index.json]: 2.x is  not needed any more

File: src/index.json [hunk 2]
```diff
@@ -275,6 +275,51 @@
...
+                        "Programming Language :: Python :: 3.4",
+                        "Programming Language :: Python :: 3.5",
+                        "Programming Language :: Python :: 3.6",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "azpycli@microsoft.com",
```
Comment (yungezz) [inline src/index.json]: wrong contact point. hi @haroldrandom  is this also auto-generated? how to fix it?

**Comment by yungezz** [comment]
> can someone trigger rerun and merge this? thank you

we'll try to merge it today, with some minor change @arrownj

**Comment by karataliu** [comment]
Thank you @yungezz . our team is waiting for this release, hope it can be addressed soon.

**Comment by arrownj** [comment]
> addressed

Hi @karataliu , could you update per Catherine commented?

File: src/index.json (hunk 2 above)
Comment (haroldrandom) [reply src/index.json]: We could discard this PR and fire another PR to remove Python 2 from variable `CLASSIFIERS` in setup.py, also change the property `author_email` in setup.py, and upgrade the patch version like `0.4.39`.
Then merge it to wait for another PR for updating index.json

**Comment by karataliu** [comment]
i did not see the comment, can you suggest where

**Comment by arrownj** [comment]
I submit another PR to update the setup.py and setup.cfg. Let's just close this one.

**Pull request closed without merging**